src/e2e/java/uk/gov/hmcts/reform/tools/Generatexml.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.hmcts.reform.tools;
+
+import static java.nio.file.FileVisitOption.FOLLOW_LINKS;
+import static java.nio.file.Files.walk;
+import static java.nio.file.Paths.get;
+import static uk.gov.hmcts.reform.tools.FileCopy.copy;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Comparator;
+import java.util.List;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.runners.MethodSorters;
+import uk.gov.hmcts.reform.tools.builders.Appeal;
+import uk.gov.hmcts.reform.tools.enums.AppealTemplate;
+import uk.gov.hmcts.reform.tools.factory.AppealFactory;
+import uk.gov.hmcts.reform.tools.utils.XmlWriter;
+
+
+@SuppressWarnings({"PMD", "checkstyle:hideutilityclassconstructor"})
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class Generatexml {
+
+    public static final String OUTPUT_DIR = "src/test/resources/updates";
+    public static final String Incoming_dir = "docker/sftp/data/incoming";
```

Comment:
this should all be capitals for `final` variable. Upper case suggests this is a class